src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,28 @@ title = "Releases"
 weight = 30
 +++
 
+## v0.8.0 Added support for OVN
+
+> This release mainly focused on adding support for interconnecting clusters that use the OVN (ovn-kubernetes) CNI plugin.
+
+Other additions and changes of note:
+
+* The active Gateway now performs periodic health checks on the connections to remote clusters and updates the Gateway
+  connection status and adds latency statistics.
+* Gateways now export the following connection metrics which can be used with Prometheus. These are currently only supported
+  for the Libreswan cable driver.
+  * The count of bytes transmitted and received between Gateways.
+  * The number of connections between Gateways and their corresponding status.
+  * The timestamp of the last successful connection established between Gateways.
+  * The RTT latency between Gateways.
+* The Libreswan cable-driver is now the default.
+* Lighthouse now prefers a Service in the local cluster first before choosing a remote cluster round-robin.
+* Lighthouse has fully migrated to use the new Kubernetes Multicluster Service CRDs (`ServiceExport` and `ServiceImport`).
+  The Lighthouse-specific CRDs are deprecated and will be removed in a future release. On upgrade from v0.7.0, exported
+  Services will automatically be migrated to the new CRDs.
+* Broker resiliency has been improved. The dataplane is no longer affected in any way if the Broker is unavailable.
```

Comment:
```suggestion
* Broker resiliency has been improved. The dataplane is no longer effected in any way if the Broker is unavailable.
```

Reply:
I believe **affected** is the right word here: **Effected** means brought about, brought into being when used as a verb. **Affected** means influenced or changed by something.

Reply:
Yes, “affected” is the correct term here.

Reply:
Okay, taking a second read of the grammer docs I understand now. Thanks guys. :+1: